### DISCUSSION TITLE
About Build The Source Code On Windows 10 Failed Issue

### DISCUSSION THREAD

**Discussion by Taoct** [memtest86plus/memtest86plus#388] [Q&A]
Now I'm try to Build the Source Code on win 10 platform with GNU and Make tools, but it Seem can not work on this platform, the report issue as the picture show
![1](https://github.com/memtest86plus/memtest86plus/assets/29203764/edd74fd0-ff08-4c08-8ce5-28e38d0224ae)
I used GNU Make 4.4 and gcc version is 14.0.0, is anyone else make success on win10?

**Comment by martinwhitaker** [comment]
Memtest86+ needs to be built using the ELF object file format. AFAIK you can only do this with GCC in Windows if you use a custom build of GCC. It might be possible with clang/llvm, but I've never tested whether Memtest86+ can be built with anything other than GCC.

**Comment by debrouxl** [comment]
Currently, Clang 16 & 18 cannot build memtest86+, but the two compile-time errors (a non-portable goto in `app/main.c`, 64-level recursivity of the `ptes64` macro in boot/startup*.S beyond the assembler's threshold of 20 levels) are easy to fix.
Building with clang produces a useful warning: see #361 .
